### DISCUSSION TITLE
support broadcast rooms which let you configure the audience to be hidden in the memberlist

### DISCUSSION THREAD

**Discussion by ara4n** [element-hq/element-meta#490] [Ideas]

**Comment by turt2live** [comment]
"like telegram" - people in hq

What this translates to is:
* No sender information on events
* No room member list
* Only group mods can post
* Default noisy notifications
* Probably a few other things

**Comment by uhoreg** [comment]
Also, possibly hide joins/parts/display name & avatar changes of normal users.  (Which may be doable in part by granular settings?)

**Comment by turt2live** [comment]
In theory if you didn't have a membership list then you wouldn't have member events. At least if this was implemented as a server-side feature (room type?). Otherwise clients would have to do a lot of filtering locally.

**Comment by KitsuneRal** [comment]
I assume this is the same as https://github.com/matrix-org/matrix-doc/issues/759?

**Comment by jfrederickson** [comment]
It's related to some of the discussion in that issue, yeah - there's another issue specifically for rooms with hidden member lists: https://github.com/matrix-org/matrix-doc/issues/951

**Comment by ara4n** [comment]
this is getting more and more desirable as time goes on, for broadcast purposes

**Comment by Ai-rin** [comment]
Recently Telegram started to show ads in channels and introduced a paid subscription to disable ads. People didn't like that, to say the least.
"Channel"-like feature would allow us to be a home for more Telegram refugees.

**Comment by natrius** [comment]
What about adding a local Broadcast group or list and just send them a message in the current existing personal room or otherwise create the personal messaging room. 
Another take and it would probably remove the need for the "Hidden members of the group" and so on. But - its a different usecase, of course.

**Comment by jswetzen** [comment]
@natrius I'm looking for exactly this to seamlessly replace a WhatsApp broadcast list. If we could get people transitioning to Matrix over time as well, that's a win.
Is there an existing issue for broadcast lists?

**Comment by vibl** [comment]
This feature has been mentioned in several issues:
- https://github.com/matrix-org/matrix-spec/issues/367
- https://github.com/matrix-org/matrix-spec/issues/249
- https://github.com/matrix-org/matrix-spec/issues/119

It seems to interest quite a few people, according to the "thumbs up" on these threads.

Is there any way it could be included in the roadmap?

FWIW, it's implemented in Rocket.Chat (they call it "broadcast channels"): https://docs.rocket.chat/guides/user-guides/rooms/channels/create-a-new-channel

**Reply by Mikaela** [reply]
In some IRCds (Unreal?) this is known as auditorium mode
